auth/store.go

```diff
@@ -290,10 +313,34 @@ func (as *authStore) Authenticate(ctx context.Context, username, password string
 		return nil, ErrAuthFailed
 	}
 
-	token := fmt.Sprintf("%s.%d", simpleToken, index)
-	as.assignSimpleTokenToUser(username, token)
+	if bcrypt.CompareHashAndPassword(user.Password, []byte(password)) != nil {
```

Comment:
Currently password checking is performed in the API layer, so this `bcrypt.CompareHashAndPassword()` is obsolete. Thanks for pointing out.